ccf_transaction_processor/scripts/ping_test.py

```diff
@@ -28,13 +28,15 @@
 ContractHome = os.environ.get("PDO_HOME") or os.path.realpath("/opt/pdo")
 CCF_Bin = os.path.join(ContractHome, "ccf", "bin")
 CCF_Etc = os.path.join(ContractHome, "ccf", "etc")
-CCF_Keys = os.path.join(ContractHome, "ccf", "keys")
+CCF_Keys = os.environ.get("PDO_LEDGER_KEY_ROOT") or os.path.join(ContractHome, "ccf", "keys")
 
 sys.path.insert(1, CCF_Bin)
+sys.path.insert(1, "../CCF/tests")
+
 from infra.clients import CCFClient
 
 # -----------------------------------------------------------------
-def ping_test(client, options, config):
+def ping_test(client, options):
     num_pings = options.num_pings
```

Comment:
config was passed in more because of the consistency of the "commands" so these files could be used as a module. not important though